web/models/assistant-model.js

```diff
@@ -87,11 +87,13 @@ class AssistantModel extends EventTarget {
 	addSourceFromBroadcastAudioURI(parsedCode) {
 		const source = {};
 
-		// First, fetch the address type
-		let addressType = BT_DataType.BT_DATA_IDENTITY;
+		// Fetch address type (AT)
 		const atToken = parsedCode.find(t => t.type === 'AT');
-		if (atToken?.value === 1) {
-			addressType = BT_DataType.BT_DATA_RPA;
+
+		// Assume 'random'/'random static' AT if not specified
+		let addressType = 1;
```

Comment:
Isn't it mandatory to specify in the URI?

Reply:
It is, but in case it's missing, I want to guesstimate for robustness :)

Reply:
Sounds like you are just presenting the user invalid sources :D I would suggest to ignore invalid values

Reply:
The likelihood of a QR code having an address but not the type... and then being a public address is low, IMO :D ... then again, this is just a dev tool for now